### PR TITLE
feat: INFRA-1686 add wss support for domains field of miniapps

### PR DESCRIPTION
**Pull request by sitozzz**
## Summary by CodeRabbit

* **New Features**
  * Added support for WebSocket (`wss://`) entries in miniapp domain configuration, with updated validation to accept `https://` and `wss://` while rejecting `http://` and `ws://`.
  * Improved miniapp domain-to-target mapping so `https://` and `wss://` for the same host share the same target index, including consistent reuse when related redirect origins establish an index.

* **Tests**
  * Expanded B2C app and miniapp resolver coverage for `wss://` handling, validation edge cases, and mapping generation/reuse rules.

### CODE REVIEW

File: apps/condo/domains/miniapp/schema/fields/domains.js [hunk 1]
```diff
@@ -53,27 +61,46 @@ const MINIAPP_DOMAINS_FIELD = {
...
-            idx++
+
+            return targetIdx === preferredIdx ? preferredIdx + 1 : preferredIdx
+        }
+
+        let idx = 2
+        for (const redirectUri of oidcRedirectURIS) {
+            if (!redirectUri) continue
+            idx = addMapping((new URL(redirectUri)).origin, idx)
         }
```
Comment (SavelevMatthew) [inline apps/condo/domains/miniapp/schema/fields/domains.js:87]: This part is broken for > 1 redirect url

**Review by SavelevMatthew — APPROVED** [review]
lgtm
